english/semantic_parsing.md

```diff
@@ -88,6 +89,15 @@ Models are evaluated based on [smatch](https://amr.isi.edu/smatch-13.pdf).
 | ChSeq + 100K (van Noord and Bos, 2017)&hearts; | 71.0 | [Neural Semantic Parsing by Character-based Translation: Experiments with Abstract Meaning Representations](https://clinjournal.org/clinj/article/view/72/64) |
 | Neural-Pointer (Buys and Blunsom, 2017)&hearts;&spades; | 61.9 | [Oxford at SemEval-2017 Task 9: Neural AMR Parsing with Pointer-Augmented Attention](http://aclweb.org/anthology/S17-2157) |
 
+### LDC2020T02
+67,507
```

Comment:
Could you correct this line? Should this be the number of examples in the dataset? According to the LDC website, the dataset only contains 59,255 examples, though.

Reply:
Yeah, you are right, sorry. I counted the number of instances with a bash command but I must have messed something up :(

Reply:
No problem at all. Thanks for the prompt reply. I've just added a suggestion below so that we have a full sentence instead of just the number of data points. :)